po/fr.po

```diff
@@ -0,0 +1,296 @@
+# French translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-09-24 23:32+0300\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. Translators: this is a format string for the zoom-percent label in the main menu.
+#. For most languages, it doesn't need translating.
+#: Window.cpp:539
+#, c-format
+msgid "%d%%"
+msgstr ""
+
+#: Window.cpp:445
+msgid "A script closed the web page"
+msgstr "Un script a fermé la page web"
+
+#: window.blp:205 window.blp:306
+msgid "Bookmarks"
+msgstr "Marque-pages"
+
+#: window.blp:51
+msgid "Bookmarks & History"
+msgstr "Marque-pages & Historique"
+
+#: Tab.cpp:215 Tab.cpp:239
+msgid "Cancel"
+msgstr "Annuler"
+
+#: window.blp:37
+msgid "Close _Other Tabs"
+msgstr "Fermer _les autres onglets"
+
+#: shortcuts-dialog.blp:63
+msgid "Close current tab"
+msgstr "Fermer l‘onglet actif"
+
+#: Tab.cpp:216
+msgid "Confirm"
+msgstr "Confirmer"
+
+#: shortcuts-dialog.blp:97 shortcuts-dialog.blp:99
+#, fuzzy
+msgid "Developer Tools"
+msgstr "Outils de développement"
+
+#: Tab.cpp:240
+msgid "Enter"
+msgstr "OK"
+
+#: js-console.blp:12
+msgid "Enter a JavaScript expression to evaluate"
+msgstr "Entrer l‘expression JavaScript à évaluer"
+
+#: Window.cpp:234
+msgid "Failed to pick file"
+msgstr "Impossible d‘ouvrir le fichier"
```

Comment:
Does "impossible" not mean "impossible", as opposed to "failed"? Is that how error messages are commonly worded in French, or?..